src/components/ClientSideLoggingToolMenu/BaseClientSideLoggingToolMenu.tsx

```diff
@@ -30,9 +32,22 @@ type BaseClientSideLoggingToolProps = {
     onDisableLogging: (logs: Log[]) => void;
     /** Action to run when enabling logging */
     onEnableLogging?: () => void;
+    /** Boolean to know if this was opened via test tools modal */
+    isViaTestToolsModal: boolean;
+    /** Action to close the test tools modal */
+    closeTestToolsModal?: () => void;
```

Comment:
We don't need to pass this in `closeTestToolsModal` prop, just import it directly `BaseClientSideLoggingToolMenu` from '@userActions/TestTool'